src/components/CountyMap/CountyMap.js

```diff
@@ -33,23 +32,22 @@ const CountyMap = ({
     });
   }, []);
 
-  const colorScale = scaleQuantile()
-    .domain(data.map(d => d.unemployment_rate))
-    .range([
-      '#ffedea',
-      '#ffcec5',
-      '#ffad9f',
-      '#ff8a75',
-      '#ff5533',
-      '#e2492d',
-      '#be3d26',
-      '#9a311f',
-      '#782618',
-    ]);
+  // TODO: Add fill back (see comment below)
```

Comment:
I have no context on this stuff. If you know if it's realistically dead, I can just delete it.